unit_tests/test_IncrementalSVDBrand.cpp

```diff
@@ -72,9 +72,9 @@ TEST(IncrementalSVDBrandTest, Test_IncrementalSVDBrand)
 
     double* basis_right_true_ans = new double[9] {
         -1.78651649346571794741E-01,     5.44387957786310106023E-01,      -8.19588518467042281834E-01,
-        -9.49719639253861602768E-01,     -3.13100149275943651084E-01,     -9.50441422536040881122E-04,
```

Comment:
This file is changed by `astyle` 2.05, but it is unchanged by 3.1. We do not have an official version of `astyle` in libROM, but I suppose it makes sense to progress to 3.1 and not revert changes back to 2.05.

Reply:
I guess you're meaning that the current PR is following astyle 3.1 convention? (Which should be, since it is enforced by ci workflow `code-style`) By the fact that the ci workflow enforces style with v3.1, we might as well say astyle v.3.1 is our official stylizer?

Reply:
Yes, actually I was confused about whether this was using 3.1 or 2.05. I think we should use 3.1 or 3.2 and make it official.